### PR TITLE
Add support for non supported vast error

### PR DESCRIPTION
**Pull request by ZacharieTFR**
### Description

- vast client will now trigger error code `102` `VAST version of response not supported.` if it parse a outdated VAST version (only 1.0 for now)
- Fix error `303` `No VAST response after one or more Wrappers` not being triggered when a wrapper containing a `<Creative>` without mediafile (e.g only with a `<CompanionAds>` inside) does not result in a VAST response.

### Issue

#476 

### Type
- [ ] Breaking change
- [ ] Enhancement
- [X] Fix
- [X] Documentation
- [ ] Tooling

### CODE REVIEW

File: docs/api/1.x/parser.md [hunk 1]
```diff
@@ -7,6 +7,8 @@ Whenever an error occurs during the VAST parsing, the parser will call on his ow
 
 * VAST error `101`: VAST schema validation error.
 
+* VAST error `102`: VAST version of response not supported.
```
Comment (clarissedmn) [inline docs/api/1.x/parser.md:10]:
```suggestion
* VAST error `102`: VAST response version not supported.
```

**Review by clarissedmn — APPROVED** [review]
LGTM 🎉

**Review by clementFrancon — APPROVED** [review]
LGTM !

File: docs/api/1.x/parser.md (hunk 1 above)
Comment (ZacharieTFR) [reply docs/api/1.x/parser.md]: Actually I edited the wrong file, this documentation is for old versions 1.x. I moved it to the correct one. The error description comes from IAB https://wiki.iabtechlab.com/index.php?title=VAST_Error_Code_Troubleshooting_Matrix

**Pull request merged**